.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,80 @@
+name: Report a problem
+description: Problem reports are for when something behaves incorrectly, or differently from how you'd expect.
+labels: [bug]
+body:
+  - type: textarea
+    id: describe
+    attributes:
+      label: Describe the problem
+      description: |
+        Provide a short description (one or two sentences) about the problem. What did you expect to happen, and what is actually happening?
+
+        If possible, provide full tracebacks (error messages) that you've encountered.
+      placeholder: |
+        The behavior of XXX doesn't work as expected. When I do YYY, I expected ZZZ to occur, but instead WWW happens.
+
+        Here's an error message I ran into...
+
+        ```python
+        momepy.Tessellation(df)
```

Comment:
The code blocks don't seem to be rendering properly here (or maybe they are?). Is this desired? See [here](https://github.com/pysal/momepy/blob/275ab12b464d4eb17273e7ed1d9e93f393faac6e/.github/ISSUE_TEMPLATE/bug_report.yml).

Reply:
Can you send a screenshot? On my side it doesn't fit the text area but that is different when you click on it in a live form.

Reply:
I'm wondering if the code snips inside the entry cells should/could be rendered differently (or if it even matters). Here is what I'm seeing:

<img width="1180" alt="Screen Shot 2022-01-11 at 8 28 45 AM" src="https://user-images.githubusercontent.com/8590583/148952100-45fe3aaf-b688-44a7-bfb4-2315875a7f6b.png">

Reply:
That is a reported bug affecting Safari only. Firefox or chrome are showing it fine. I'd say we let Github fix it.